HW7/dist/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
You shouldn't pul all the application's logic in the html file. It should be like a template, how to generate html for your project, like this:
```<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>ToDo</title>
    
</head>
<body>
<div id="root"></div>
</body>

</html>